src/NovelRT/Graphics/TextRect.cpp

```diff
@@ -152,4 +155,9 @@ namespace NovelRT::Graphics {
       ptr->setActive(false);
       });
   }
+
+  void TextRect::setActive(bool value) {
+    WorldObject::setActive(value);
+    std::for_each(_letterRects.begin(), _letterRects.end(), [&value](auto& x){x->setActive(value);});
```

Comment:
Do we need to set each letter as active? Shouldn't the textrect being active/inactive impact the individual letters at a higher point (such that leaving the letters always active is fine)